kernel/kernel-core/src/main/java/io/mosip/kernel/core/util/CryptoUtil.java

```diff
@@ -99,26 +99,41 @@ public static String encodeBase64String(byte[] data) {
 	 */
 	@Deprecated(since = "1.1.5.5", forRemoval = true)
 	public static byte[] decodeBase64(String data) {
+		if (EmptyCheckUtils.isNullEmpty(data)) {
+			return null;
+		}
 		try {
-			return Base64.getUrlDecoder().decode(data.getBytes());
+			return Base64.getUrlDecoder().decode(data);

```

Comment:
why getBytes is not called for this, but called for decodeURLSafeBase64 method ?

Reply:
removed all get bytes